packages/installer/deploy.sh

```diff
@@ -0,0 +1,233 @@
+#!/bin/bash
+set -e
+
+CHANNEL_UPGRADES="${CHANNEL_UPGRADES:-true}"
+FORCE=""${FORCE:-false}
+
+# 1. Identify active/passive partitions
+# 2. Deploy active and passive to the same image
+
+find_partitions() {
```

Comment:
everything in here seems very similar to cos-installer. Could we extract most of this functions into a cos-functions file and then source it in both scripts?

Reply:
that sounds good, but I would rather do in a separate follow-up PR. The same applies for `cos-upgrade`, `cos-installer` and `cos-reset`

Reply:
Yes I agree I would do it in a separate PR or even leave it as is until we start working on #108 IMHO all this scripts should be substituted by a client in the scope of #108. Probably a fast solution is to build a shell client so we can reuse many portions of the code and we not have to start from scratch, I am not very passionate about this option, but probably this is a good way to start splitting all these logic into pieces and compose some client skeleton.

Reply:
Indeed I am not very happy of this `cos-deploy` script, but I though that abusing of `cos-upgrade` to also cover this use case was making the behavior far from obvious (very small code changes were needed, but then the _upgrade_ behavior looked pretty strange to my eyes). That's why I opted for creating its own script by just copying the `cos-upgrade` code, removing many parts and applying few logic changes. I duplicated quite some code, but since this is already the case for all other `cos-*` scripts I assumed we can go with it for now and once the images and CI stories are addressed raise #108 prio